modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/utils/numberShorten.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+/*
+ * If a number is bigger than 10000 it will transform into thousands (K).
+ * If a number is bigger than 1000000 it will transform into millions (M).
+ * If a number is bigger than 1000000000 it will transform into billions (B).
+ *
+ * 4 => 4
+ * 4000 => '4K'
+ * 4000000 => '4M'
+ * 4000000000 => '4B'
+ */
+export function numberShorten(value) {
+	if (value >= 1000000000) {
```

Comment:
For readability and to reduce the chances of making a mistake, you might be able to write `1_000_000_000` here (check though; off the top of my head, I don't think we have [babel-plugin-proposal-numeric-separator](https://babeljs.io/docs/en/babel-plugin-proposal-numeric-separator) explicitly enabled, even though we could because [the proposal](https://github.com/tc39/proposal-numeric-separator) is at stage 3; but it's something we should definitely enable).

I created https://github.com/liferay/liferay-npm-tools/issues/470 as a reminder to turn this plugin on by default, if it isn't already.

Reply:
I tried that but I have a babel error. Should I enable something to make this change or should I wait until it' definitely enabled? 🤔

Reply:
Yeah, just leave it and one day it will magically start working. When we add it we will do a grep (`git grep '\b\d{5,}' -- '*.js'`) looking for long-ish numbers to update to use the new syntax.

Reply:
Great! Thanks! 😄